### ISSUE TITLE
2.2 T1

### ISSUE THREAD

**Issue by Nicolas413** [IIC2440/Syllabus-2024-1#6]
Hola, tengo una consulta referente a los datos, consideraneo que usuario_ordenes_x, tiene compra_id y las ordenes_x tienen ordenes_id, estas dos id´s son las mismas no?, por que sino no se puede saber que compro un usuario si es que son distintas.

**Comment by alanezz** [comment]
Sí, órdenes_id es compatible con compra_id. Aunque recuerda que estos campos no son únicos en el documento, porque se resetean mes a mes.
